tools/ThermoRawFileParser/thermo_converter.xml

```diff
@@ -1,144 +1,201 @@
     <command>
 <![CDATA[
 #import re
 
-mkdir ./raws_folder &&
-mkdir ./output_folder &&
-#for $input_raw in $input_files:
-    #set $input_name = re.sub('[^\w\-\.]', '_',$input_raw.element_identifier.split('/')[-1].replace(".raw", "") + ".raw")
-    ln -s -f '${input_raw}' './raws_folder/${input_name}' &&
-#end for
+#set $input_name = re.sub('[^\w\-\.]', '_', $input_file.element_identifier.replace(".raw", "") + ".raw")
+ln -s -f '$input_file' '$input_name' &&
 
 ThermoRawFileParser.sh
-    -d=./raws_folder
-    -o=./output_folder
-    -f=$output_format
+    --input='$input_name'
+    --output_file='$output'
+    -f=$format_cond.output_format
     #if $output_metadata_selector != "off":
-        --metadata="${output_metadata_selector}"
+        --metadata="$output_metadata_selector"
+        --metadata_output_file='$output_metadata'
+    #end if
+    #if $format_cond.output_format == "0"
+        $format_cond.mgfPrecursor
+    #end if
+    #if $format_cond.output_format in ['1', '2']
+        $format_cond.zlib_boolean
+        $format_cond.peakpicking_boolean
     #end if
-    $zlib_boolean
-    $peakpicking_boolean
     $ignore_instrument_errors_boolean
-
+    $allDetectors
+    $includeExceptionData
+    #if $msLevel != ''
+        --msLevel="$msLevel"
+    #end if
 ]]>
     </command>
     <inputs>
-        <param name="input_files" type="data" format="thermo.raw" label="Thermo RAW file" help="" multiple="true"
-          optional="False" />
-
-        <param name="output_format" type="select" label="Output format">
-            <option value="0">mgf</option>
-            <option value="1" selected="True">mzml</option>
-            <option value="2">Indexed mzml</option>
-        </param>
-
-        <param name="zlib_boolean" type="boolean" truevalue="" falsevalue="-z" checked="true"
-            label="Use zlib compression for the m/z ratios and intensities" help="" />
-
-        <param name="peakpicking_boolean" type="boolean" truevalue="" falsevalue="-p" checked="true"
-            label="Use the peak picking provided by the native thermo library" help="" />
-
-        <param name="ignore_instrument_errors_boolean" type="boolean" truevalue="-e" falsevalue="" checked="true"
-            label="Ignore missing instrument properties" help="If false, it stops the conversion if instrument properties are missing" />
-
-        <param name="output_metadata_selector" type="select" label="Output metadata" >
+        <param argument="--input" name="input_file" type="data" format="thermo.raw" label="Thermo RAW file" help="" optional="False"/>
+        <conditional name="format_cond">
+            <param argument="--format" name="output_format" type="select" label="Output format">
+                <option value="0">mgf</option>
+                <option value="1" selected="True">mzml</option>
+                <option value="2">Indexed mzml</option>
+                <option value="3">Parquet</option>
```

Comment:
Is there a more specific name for it? Parquet is like hdf5, a general concept/format.

Reply:
Readme says _Apache Parquet: under development_ .. see also https://github.com/compomics/ThermoRawFileParser/issues/19 :) .. so maybe we skip this and choose profile 20.05 or 20.09?

Reply:
@bernt-matthias That seems like the better choice imho, wrapping unfinished features is tricky.

Reply:
I disabled parquet output for now.

Reply:
For future re-enabling the format they used seems to be implemented here: https://github.com/compomics/ThermoRawFileParser/blob/c2d4e16347262074c8c998768f306f1ed7043191/Writer/ParquetSpectrumWriter.cs#L134-L153